tests/suite.conf

```diff
@@ -0,0 +1,11 @@
+[General]
+timeout: 0.1
+
+[paths]
+exiv2: ../build/bin/exiv2
```

Comment:
Yeah, I know. Combined with the fact that the config file's path is currently hardcoded, makes it really inconvenient to change the paths here (as you'd have to edit the file that is under version control).

I was thinking about adding a section `[ENV]` to the config files where you could specify which variables you would like to extract from the environment variables. And then you could use configparsers interpolation to make it more dynamic. This could also fix the missing extension on windows. This could look like this for example:
``` ini
[ENV]
exiv2_path: EXIV2_PATH
exiv2_file_type: EXIV2_FILE_TYPE

[paths]
exiv2: ${ENV:exiv2_path}/exiv2${ENV:exiv2_file_type}
```
If an environment variable is unset, then it would expand to "". If we then provide some defaults in the `[DEFAULT]` section then this could work.